Conference archive & Authors / OSCI / Speakers as community members

astephanus opened the pull request:
### Description

* Community members listing page.
* Community members individual pages.
* Migration of Authors, OSCI participants, and Conference Speakers as Community Members with corresponding User Personas with support for multiple personas per member; for example some Authors are also Conference Speakers.
  * Normalization of Front Matter properties.
  * Deduplication and resolution of conflicting `short_name` properties.
  * New `_data` collection for configurable naming of
* OpenSearch Conference reorganization, configuration, and renaming of collections and templates for generic use for either current / upcoming conference content as well as past / archived conference content with identified by `conference_id`.
* Archival of 2022, and 2023 OpenSearch Conferences.
* Navigation menu changes for conference archival as well as support for another level of depth for navigation menu nesting.
* Navigation menu template changes to support run-time determination of menu item highlighting that reduces template complexity and ensures a more future proof implementation of this feature.
* New `_sample.md` files to document the expectations for new collection items.
* New reusable content filtering widget repurposed from the existing Events pages which is also reused on the new Community Member Listing page, and will be used on the upcoming new Blog Listing page, and new Partners listing pages, as well as wherever else it may be needed in the future.
* Missing interactive card and button states on existing platform pages.
* New reusable page title and breadcrumbs include template shared across layouts.
* Various breadcrumb fixes.
* Various CSS fixes.
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff


By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

krisfreedain commented:
this is looking incredible @astephanus 

As discussed on Slack, please go ahead and move the OSCI group offline for now while the data goes through cleanup, and the small change to ensure the lorem ipsum is updated on the 2022/2023 archive pages to the appropriate text

astephanus commented:
> this is looking incredible @astephanus
> 
> As discussed on Slack, please go ahead and move the OSCI group offline for now while the data goes through cleanup, and the small change to ensure the lorem ipsum is updated on the 2022/2023 archive pages to the appropriate text

I prepended an underscore to the OSCI participants directory which omits them from the build, and I also removed the OSCI option from the persona filter configuration data file. I added the copy discussed in Slack to those conference archive landing pages.

krisfreedain reviewed (approved):
Amazing. Thank you @astephanus
